gateway/ln-gateway/proto/gateway_lnrpc.proto

```diff
@@ -30,6 +32,21 @@ service GatewayLightning {
   rpc CompleteHtlc(InterceptHtlcResponse) returns (EmptyResponse) {}
 
   rpc CreateInvoice(CreateInvoiceRequest) returns (CreateInvoiceResponse) {}
+
+  /* Connect the underlying lightning node to another node. */
+  rpc ConnectToPeer(ConnectToPeerRequest) returns (EmptyResponse) {}
+
+  /* Get a Bitcoin address that belongs to the underlying lightning node's wallet. */
+  rpc GetFundingAddress(EmptyRequest) returns (GetFundingAddressResponse) {}
+
+  /* Open a channel on the underlying lightning node. */
+  rpc OpenChannel(OpenChannelRequest) returns (EmptyResponse) {}
```

Comment:
PR Review Club: Should we document `OpenChannel` and `GetFundingAddress` as required (and mark the rest as optional)?